src/routes/user/deleteUser.ts

```diff
@@ -0,0 +1,61 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { Type } from '@sinclair/typebox';
+import { knexController } from '../../database/database';
+import {
+  userIdParamsPayload,
+  UserIdParamsPayload,
+} from '../../types/user.types';
+
+const deleteUserReply = Type.Object({
+  userId: Type.String(),
```

Comment:
```suggestion
  userId: Type.String({format: 'uuid'}),
```
Perusteluja:
* Nyt swagger auttaa fronttidevaajaa
* Virheviesti auttaa fronttidevaajaa
* Selkeän virheelliset (vihamieliset 😨 ) pyynnöt ei pääse tietokantakyselyiksi asti

Reply:
Tärkeämpää olisi tietysti tehdä sama muutos tuohon UserIdParamsPayload-tyyppiin